examples/aws/rds-s3/README.md

```diff
@@ -0,0 +1,148 @@
+
+```sh
+while ! kustomize build . | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
+Once, everything is installed successfully, you can access the Kubeflow Central Dashboard [by logging in to your cluster](../../../README.md#connect-to-your-kubeflow-cluster).
+
+Congratulations! You can now start experimenting and running your end-to-end ML workflows with Kubeflow.
+
+### Install individual components
+
+#### Kubeflow Pipelines with RDS and S3
+
+Make sure you have followed the steps at [Create RDS Instance](#create-rds-instance) to prepare your RDS MySQL database for integration with Kubeflow Pipelines. 
+
+Make sure you have also followed the steps at [Create S3 Bucket](#create-s3-bucket) to prepare your S3 for integration with Kubeflow Pipelines. 
+
+1. Go to the pipelines manifest directory `<KUBEFLOW_MANIFESTS_REPO_PATH>/apps/pipeline/upstream/env/aws`
+```
+cd <KUBEFLOW_MANIFESTS_REPO_PATH>/apps/pipeline/upstream/env/aws/
+```
+
+2. Configure `params.env` with the RDS endpoint URL, S3 bucket name, and S3 bucket region that were configured when following the steps in [Create RDS Instance](#create-rds-instance) and [Create S3 Bucket](#create-s3-bucket). 
+
+- For example if your RDS endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, S3 bucket name is `kf-aws-demo-bucket`, and s3 bucket region is `us-west-2` your `params.env` file should look like:
+
+```
+dbHost=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+
+bucketName=kf-aws-demo-bucket
+minioServiceHost=s3.amazonaws.com
+minioServiceRegion=us-west-2
+```
+
+3. Configure `secret.env` with your RDS database username and password that were configured when following the steps in [Create RDS Instance](#create-rds-instance). 
+
+- For example if your username is `admin` and your password is `Kubefl0w` then your `secret.env` file should look like:
+
+```
+username=admin
+password=Kubefl0w
+```
+
+4. Configure `minio-artifact-secret-patch.env` with your AWS credentials.
+
+Find more details about configuring/getting your AWS credentials here:
+https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
+
+```
+accesskey=AXXXXXXXXXXXXXXXXXX6
+secretkey=eXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXq
+```
+
+5. [Single component installation] Install
+
+```
+cd <KUBEFLOW_MANIFESTS_REPO_PATH>
+kustomize build apps/pipeline/upstream/env/aws/ | kubectl apply -f -
+
+# If upper one action got failed, e.x. you used wrong value, try delete, fix and apply again
+kustomize build apps/pipeline/upstream/env/aws/ | kubectl delete -f -
+```
+
+#### Katib with RDS
+
+Make sure you have followed the steps at [Create RDS Instance](#create-rds-instance) to prepare your RDS MySQL database for integration with Kubeflow Pipelines. 
+
+1. Go to the katib manifests directory `apps/katib/upstream/installs/katib-external-db-with-kubeflow`
+```
+cd <KUBEFLOW_MANIFESTS_REPO_PATH>/apps/katib/upstream/installs/katib-external-db
```

Comment:
shall we use printf command to save a few steps and reduce the reading for the user?
```
printf 'KATIB_MYSQL_DB_DATABASE=KubeflowRDS1
KATIB_MYSQL_DB_HOST=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
KATIB_MYSQL_DB_PORT=3306
DB_USER=admin
DB_PASSWORD=Kubefl0w
' > apps/katib/upstream/installs/katib-external-db/secrets.env
```
same for other places

Reply:
i think having printf would result in the user copying and pasting rather than verifying the parameters they are inputting, thinking that the given parameters are the ones to use

Reply:
ok for now, can iterate later